src/test/java/org/heigit/ohsome/ohsomeapi/controller/PostControllerTest.java

```diff
@@ -50,6 +50,31 @@ public static void applicationMainShutdown() {
     }
   }
 
+  /*
+   * test geometries lying outside the underlying data-extract polygon
+   */
+
+  @Test
+  public void providedBcirclesOutsideUnderlyingDataExtractPolygonTest() {
+    TestRestTemplate restTemplate = new TestRestTemplate();
+    MultiValueMap<String, String> map = new LinkedMultiValueMap<>();
+    map.add("bcircles", "8.457261,49.488483,100");
+    ResponseEntity<JsonNode> response =
+        restTemplate.postForEntity(server + port + "/users/count", map, JsonNode.class);
+    assertEquals(404, response.getBody().get("status").asInt());
```

Comment:
Do you want to check for the correct message as well? Or do you think the status code is sufficient?

Same question for the next test.